src/server/coprocessor/mod.rs

```diff
@@ -62,6 +64,18 @@ impl From<engine::Error> for Error {
     }
 }
 
+impl From<xeval::Error> for Error {
```

Comment:
the coprocessor Error enum has an Other member which auto impl From trait func by quick_error crate.